app.js

```diff
@@ -0,0 +1,18 @@
+const express = require('express')
+const { controllerSetUp, invalidPath } = require('./controllers/api.controller')
+const { getTopics } = require('./controllers/topics.controller')
+const { handleServerErrors, handlePSQLErrors } = require('./errors')
+
+const app = express()
+
+app.use(express.json())
```

Comment:
Please don't add code that is not yet being used 😄 (it's bad practise as it can be confusing to other devs/ your future self)